core/src/main/scala/cats/instances/eq.scala

```diff
@@ -8,7 +8,7 @@ trait EqInstances {
        * Defaults to the trivial equivalence relation
        * contracting the type to a point
        */
-      def unit[A]: Eq[A] = Eq.allEqual
+      def unit: Eq[Unit] = Eq.allEqual
```

Comment:
not quite sure I understand why this one is reported as not tested either. Any idea?